### PR TITLE
[MLR] Re-order MLR Entry metadata and table headers in all instances

### PR DESCRIPTION
**Pull request by ajaitasaini**
### Related ticket(s)

MDCT-2566

NOTE: this PR also includes updates to the readme for an issue + workaround for DynamoDB issues during setup

---
### How to test

- Login
- Add/create + enter MLR submission
- Navigate to "MLR Reporting" tab on left
- Click "Add program reporting information" button
- Fill + save reporting info
- Confirm the following changes:
    - image
- Click "Enter MLR" button 
- Confirm the following changes:
    - image
- Scroll to bottom, click "Save and Return" button
- Navigate to "Review and Submit" tab on left
- Click "Review PDF" button
- Confirm that the same UI changes are in the PDF

### Important updates

---
### Author checklist

- [x] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [x] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Comment by ailZhou** [comment]
Looks great in the forms! I think one part of the export page might be missing the change, under program reporting information?

Screenshot 2023-06-23 at 9 55 39 AM

**Comment by karla-vm** [comment]
> Looks great in the forms! I think one part of the export page might be missing the change, under program reporting information?
> 
> Screenshot 2023-06-23 at 9 55 39 AM

I'm seeing that field in the PDF missing as well! On the form itself, everything looks good 💯

**Comment by ajaitasaini** [comment]
@ailZhou @karla-vm thank you for the catch!! should be fixed now

**Review by karla-vm — APPROVED** [review]
lookin' good!

**Review by ailZhou — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -87,6 +87,11 @@ Once the update ships [in dynamodb-localhost](https://github.com/99x/dynamodb-lo
 and [in serverless-dynamodb-local](https://github.com/99x/serverless-dynamodb-local/issues/294), the override in `./package.json` and
 this section of the README will become obsolete and should be removed.
 
+#### DynamoDB Local failed to start with code 1 
+If you're getting the above error and/or `inaccessible host: 'localhost' at port '8000'`, some steps to try:
+- confirm that you're on the right Java version -- if you have an M1 mac, you need an [x86 install](https://www.azul.com/downloads/?version=java-18-sts&os=macos&architecture=x86-64-bit&package=jdk#zulu) 
+- delete your `services/database/.dynamodb` directory and then run `dev local` in your terminal
+
```
Comment (benmartin-coforma) [inline README.md:94]: Praise: I love to see these helpful tips in the readme!

File: services/ui-src/src/components/overlays/EntityDetailsOverlay.tsx [hunk 2]
```diff
@@ -214,7 +214,7 @@ const sx = {
         whiteSpace: "break-spaces",
         fontSize: "xl",
         lineHeight: "1.75rem",
-        "&:last-child": {
+        "&:first-child": {
```
Comment (benmartin-coforma) [inline services/ui-src/src/components/overlays/EntityDetailsOverlay.tsx:217]: Praise: Great catch!

**Review by benmartin-coforma — APPROVED** [review]
The code looks great!

**Pull request merged**